pages.es/common/nmap.md

```diff
@@ -1,37 +1,37 @@
+> Algunas funciones (p. ej. escaneo SYN) se activan solamente cuando `nmap` es ejecutado con los permisos del superusuario.
 > Más información: <https://nmap.org/book/man.html>.
 
-- Comprueba si una dirección IP está activa y adivina el sistema operativo del host remoto:
+- Escanea los top 1000 puertos de un host remoto con varios niveles de [v]erbosidad:
 
-`nmap -O {{ip_o_nombre_del_host}}`
+`nmap -v{{1|2|3}} {{ip_o_nombre_de_host}}`
 
-- Intenta determinar si los hosts especificados están activos (exploración ping) y cuáles son sus nombres y direcciones MAC:
+- Ejecuta un barrido de ping sobre una subred o hosts específicos:
 
-`sudo nmap -sn {{ip_o_nombre_del_host}} {{opcional_otra_dirección}}`
+`nmap -T5 -sn {{192.168.0.0/24|ip_o_nombre_de_host1,ip_o_nombre_de_host2,...}}`
 
-- Habilita también los scripts, la detección de servicios, la huella digital del SO y el traceroute:
+- Activa la detección de sistemas operativos, la detección de versión, el escaneo con guiones y traceroute:
 
-`nmap -A {{dirección_o_direcciones}}`
+`sudo nmap -A {{ip_o_nombre_de_host1,ip_o_nombre_de_host2,...}}`
 
-- Escanea una lista específica de puertos (usa '-p-' para todos los puertos desde 1 al 65535):
+- Escanea una lista de puertos (invoca `nmap` con `-p-` para escanea todos los puertos desde 1 a 65535):
 
-`nmap -p {{port1,port2,...,portN}} {{dirección_o_direcciones}}`
+`nmap -p {{puerto1,puerto2,...}} {{ip_o_host1,ip_o_host2,...}}`
 
-- Realiza la detección de servicios y versiones de los 1000 puertos principales utilizando los scripts por defecto de NSE; escribiendo los resultados ('-oN') en el fichero de salida:
+- Detecta el servicio y versión de los top 1000 puertos usando los guiones NSE por defecto y escribe los resultados (`-oA`) en archivos de salida:
 
-`nmap -sC -sV -oN {{top-1000-puertos.txt}} {{dirección_o_direcciones}}`
+`nmap -sC -sV -oA {{top-1000-ports}} {{ip_o_host1,ip_o_host2,...}}`
 
-- Escanea objetivo(s) cuidadosamente utilizando los scripts NSE "default and safe":
+- Escanea objetivo(s) cuidadosamente usando los guiones NSE `default and safe`:
 
-`nmap --script "default and safe" {{dirección_o_direcciones}}`
+`nmap --script "default and safe" {{ip_o_host1,ip_o_host2,...}}`
 
-- Escanea el servidor web que se ejecuta en los puertos estándar 80 y 443 utilizando todos los scripts de NSE 'http-*' disponibles:
+- Escanea servidores web ejecutandose en los puertos estándares 80 y 443 usando todos los guiones `http-*` NSE disponibles:
 
-`nmap --script "http-*" {{dirección_o_direcciones}} -p 80,443`
+`nmap --script "http-*" {{ip_o_host1,ip_o_host2,...}} -p 80,443`
 
-- Realiza un escaneo sigiloso muy lento ('-T0') intentando evitar la detección por parte de IDS/IPS y utiliza direcciones IP de origen con señuelo ('-D'):
+- Intenta evadir los sistemas de detección y prevención de intrusos escaneando extremadamente lento (`-T0`) y usando direcciones de origen de señuelo (`-D`), paquetes [f]ragmentados, datos aleatorios y otros métodos:
```

Comment:
escaneando de una manera extremadamente lenta

Reply:
Why? Your suggested description is longer.